_targets.R

```diff
@@ -0,0 +1,45 @@
+library(targets)
+source("code.R")
```

Comment:
It will not affect this training to use the code.R vs. src sourcing. You can check out the issue I filed [here](https://github.com/USGS-R/ds-pipelines-targets-1-course/issues/17). I think the text does imply creating the code.R file